frontend/packages/dev-console/src/components/AddPage.tsx

```diff
@@ -1,17 +1,49 @@
 import * as React from 'react';
 import { Helmet } from 'react-helmet';
+import { match as RMatch } from 'react-router';
+import { history } from '@console/internal/components/utils';
+import { createProjectModal } from '@console/internal/components/modals';
 import ODCEmptyState from './EmptyState';
 import NamespacedPage from './NamespacedPage';
+import DefaultPage from './DefaultPage';
 
-const AddPage: React.FC = () => (
-  <React.Fragment>
-    <Helmet>
-      <title>+Add</title>
-    </Helmet>
-    <NamespacedPage>
-      <ODCEmptyState title="+Add" />
-    </NamespacedPage>
-  </React.Fragment>
-);
+export interface AddPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+const openProjectModal = () =>
+  createProjectModal({
+    blocking: true,
+    onSubmit: (project) => history.push(`/add/ns/${project.metadata.name}`),
+  });
+const AddPage: React.FC<AddPageProps> = ({ match }) => {
+  const namespace = match.params.ns;
+  return (
+    <React.Fragment>
+      <Helmet>
+        <title>+Add</title>
+      </Helmet>
+      <NamespacedPage>
+        {namespace ? (
+          <ODCEmptyState title="+Add" />
+        ) : (
+          <DefaultPage title="Add">
+            <div>
+              Select a project to start adding to it or
+              <button
+                type="button"
+                className="btn btn-link btn-link--no-padding"
```

Comment:
`btn-link--no-padding` doesn't seem to do anything.
The button alignment looks off:
![image](https://user-images.githubusercontent.com/14068621/60319833-b3cf2100-9945-11e9-9d79-685227957079.png)

Reply:
> `btn-link--no-padding` doesn't seem to do anything.
> The button alignment looks off:
> ![image](https://user-images.githubusercontent.com/14068621/60319833-b3cf2100-9945-11e9-9d79-685227957079.png)

We use `btn-link--no-btn-default-values` (although I'd like to rename it).

Looks like we have another `btn-link--no-padding` we should fix:

packages/dev-console/src/components/source-to-image/SourceToImage.tsx
324:                    className="btn btn-link btn-link--no-padding"

Reply:
used `btn-link--no-btn-default-values` to fix the alignment issue.
![screenshot-localhost-9000-2019 06 29-00-53-23](https://user-images.githubusercontent.com/9964343/60366177-57810580-9a08-11e9-9711-ff5d8ab827ab.png)